draft-vvv-webtransport-http3.md

```diff
@@ -157,10 +157,61 @@ is closed.
 
 ## Establishing a Transport-Capable HTTP/3 Connection
 
+In WebTransport, an *exclusive mode* is a state of HTTP/3 connection where the
+connection is expected to have exactly one WebTransport session, and no other
+HTTP requests.
+
 In order to indicate support for WebTransport, both the client and the server
-MUST send a SETTINGS_ENABLE_WEBTRANSPORT value set to "1" in their SETTINGS
-frame.  Endpoints MUST NOT use any WebTransport-related functionality unless
-the parameter has been negotiated.
+MUST send a SETTINGS_ENABLE_WEBTRANSPORT setting.  The default value of
+SETTINGS_ENABLE_WEBTRANSPORT is "0" (Unsupported), however it can also assume
+values of "1" (Supported) and "2" (Exclusive).  The exact meaning of the value
+depends on the party.
+
+SETTINGS_ENABLE_WEBTRANSPORT sent by the client has the following possible values:
+* Unsupported ("0") indicates that the client does not support WebTransport
+  over HTTP/3.
+* Supported ("1") indicates that the client supports WebTransport over HTTP/3
+  and may potentially initiate a WebTransport session in the future.
+* Exclusive ("2") indicates that the client both supports WebTransport over
+  HTTP/3 and has a WebTransport request that it intends to send over the
+  connection in question.
```

Comment:
Does this mean the client has to have a request in mind before sending the SETTING?  Is pre-connecting allowed?

Reply:
Sure, as long as you're fine with the resulting connection being usable for a single WebTransport.